src/signals/settings/users/Detail/components/UserForm/__tests__/UserForm.test.js

```diff
@@ -38,6 +38,9 @@ describe('signals/settings/users/containers/Detail/components/UserForm', () => {
     expect(container.querySelectorAll('[name="is_active"]')[0].checked).toBe(true);
     expect(container.querySelectorAll('[name="is_active"]')[1].value).toBe('false');
     expect(container.querySelectorAll('[name="is_active"]')[1].checked).toBe(false);
+
+    expect(container.querySelectorAll('[name="departments"]')).toHaveLength(18);
+    expect(container.querySelectorAll('[name="departments"]')[7].checked).toBe(false);
```

Comment:
Can the index be inferred from the values in `departmentsJson`?

Reply:
Done